codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerMessageDeflateDecoderTest.java

```diff
@@ -58,19 +63,18 @@ public void testCompressedFrame() {
         // test
         assertNotNull(uncompressedFrame);
         assertNotNull(uncompressedFrame.content());
-        assertTrue(uncompressedFrame instanceof BinaryWebSocketFrame);
```

Comment:
Why was this removed ?

Reply:
https://github.com/netty/netty/blob/721d2c02da639bcc61cd1659921591018a4c7f3d/codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerMessageDeflateDecoderTest.java#L61